### ISSUE TITLE
Add recommendation for thumbnails in referenced collections

### ISSUE THREAD

**Issue by Fiser33** [IIIF/api#1502]
As a developer of mobile Presentation API client I'd like to propose addition of simple recommendation along with brief explanation to the Presentation API specification. The recommendation is that collection references are recommended to contain collection thumbnail if present. The reason is following:

_Client sends request for a collection and gets list of references to other collections. To enhance user experience, client is required to display a thumbnail for each item, but references only require to contain link and title. Displaying thumbnail therefore means to send at least one additional request for each item. If thumbnail is missing, client displays first image resource from item's content. In case of collections this leads to DFS like algorithm. This may result in lot of network requests just in order to display single thumbnail._

Typical response as for Presentation API 2.1.1:
```
{
  "@context": "http://iiif.io/api/presentation/2/context.json",
  "@id": "http://www.example.org/path/to/collection/top",
  "@type": "sc:Collection",
  "label": "Top level collection",

  "members": [
    {
      "@id": "http://www.example.org/path/to/collection/one",
      "@type": "sc:Collection",
      "label": "Collection one"
    },
    ...
  ]
}
```

Proposed response:
```
{
  "@context": "http://iiif.io/api/presentation/2/context.json",
  "@id": "http://www.example.org/path/to/collection/top",
  "@type": "sc:Collection",
  "label": "Top level collection",

  "members": [
    {
      "@id": "http://www.example.org/path/to/collection/one",
      "@type": "sc:Collection",
      "label": "Collection one",
      "thumbnail": "http://www.example.org/path/to/thumbnail_1.png"
    },
    ...
  ]
}
```

**Comment by sdellis** [comment]
👍 

The overhead of making an http request for every object in a collection is not worth it for a thumbnail that, according to the spec, _SHOULD_ be included. _Performant_ clients should assume that if a thumbnail is not included in the members list object, then it doesn't exist.

**Comment by tomcrane** [comment]
Include in example in Prezi 5.1
Remove "Typically..." from http://prezi3.iiif.io/api/presentation/3.0/#terminology
...and describe use of thumbs in more detail in applicable resources, such as Collections.

**Comment by azaroth42** [comment]
Closed by #1525
